### PR TITLE
Fix/boas 694 fix upload download

### PR DESCRIPTION
**Pull request by lutamKC**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/lib/active-directory-token.js [hunk 1]
```diff
@@ -19,22 +18,18 @@ const getActiveDirectoryAccessToken = async (session) => {
 		throw new Error('Session account not found');
 	}
 
-	/** @type {AuthenticationResult | null} * */
+	/** @type {{accessToken: string, expiresOn: any} | null} * */
 	const blobResourceAuthResult = await acquireTokenSilent(sessionAccount, [
 		'https://storage.azure.com/user_impersonation'
 	]);
 
-	if (!blobResourceAuthResult?.accessToken || !blobResourceAuthResult?.expiresOn) {
```
Comment (lutamKC) [inline apps/web/src/server/lib/active-directory-token.js:27]: expiresOn can actually be null

File: apps/web/src/server/app/components/file-uploader.component.js [hunk 2]
```diff
@@ -30,34 +27,16 @@ export async function postDocumentsUpload({ params, body, session }, response) {
 	const uploadInfo = await createNewDocument(caseId, body);
 	const { documents } = uploadInfo;
 
-	const sessionAccount = getAccount(session);
+	const accessToken = await getActiveDirectoryAccessToken(session);
```
Comment (lutamKC) [inline apps/web/src/server/app/components/file-uploader.component.js:30]: the real fix: I forgot the AWAIT

**Review by mayya-bond-kc — APPROVED** [review]

**Review by francescobrenci-AR — APPROVED** [review]

**Pull request merged**